typings/internal-ambient.d.ts

```diff
@@ -58,6 +58,7 @@ declare namespace NodeJS {
     body: Uint8Array | BodyFunc;
     session?: Electron.Session;
     partition?: string;
+    referrer?: string;
```

Comment:
This change is no longer needed.

Reply:
But I got build failures without this change.

Reply:
@jkleinsc this is in the internal typings

Reply:
I have added this line back to fix the build failure.